standard/namespaces.md

```diff
@@ -166,6 +167,7 @@ Within global attributes and member declarations in a compilation unit or namesp
 > namespace N3
 > {
 >     using A = N1.N2.A;
+>
```

Comment:
@jskeet This was one type of change I wanted to check on. I think it's better by having the blank line, but it's a very mild preferences. 

For our "coding style", I'd like to say different elements should have a blank line between them. On the other hand, I like a rule that doesn't add a blank line where the element is an empty class or namespace.

See also line 179, 192, 201-203.

Reply:
Mmm, interesting. I probably wouldn't have added blank lines in 179, 192, 201-203... but I'm quite happy for us to do so, and that should probably be the default. We should just allow ourselves to have an exception for things like:

```csharp
public void M(int x) {}
public void M(string y) {}
public void M(int x, string y = null) {}
```

... in an example about overload resolution, for example.

Reply:
I agree. I think that's easily handled by adding methods to my exception in the previous comment.

Reply:
I opened #358 to start the coding standard discussion. 

I'll merge this.